Dockerfile

```diff
@@ -19,21 +21,28 @@ EXPOSE 5432
 EXPOSE 8000
 EXPOSE 6060
 EXPOSE 6061
+EXPOSE 8002
 EXPOSE 11625
 EXPOSE 11626
 
 ADD dependencies /
 RUN /dependencies
 
 COPY --from=stellar-xdr /usr/local/cargo/bin/stellar-xdr /usr/local/bin/stellar-xdr
-
 COPY --from=stellar-core /usr/local/bin/stellar-core /usr/bin/stellar-core
-
 COPY --from=horizon /go/bin/horizon /usr/bin/stellar-horizon
-
 COPY --from=friendbot /app/friendbot /usr/local/bin/friendbot
-
 COPY --from=stellar-rpc /bin/stellar-rpc /usr/bin/stellar-rpc
+COPY --from=lab /lab /opt/stellar/lab
+COPY --from=lab /usr/local/bin/node \
```

Comment:
I thought the lab is a static website. Can we build the static site and host it with nginx instead?

Reply:
I don't think it is, but I asked in Slack anyway.